nixos/lib/test-driver/test-driver.py

```diff
@@ -642,6 +642,14 @@ def wait_for_x(self):
                 if status == 0:
                     return
 
+    def copy_file_from_host(self, source, target):
+        """Copies the file located at source path to the target path.
+        Note: "source" is equivalent to "from", "target" is equivalent to "to" in the Perl script
```

Comment:
Comments that reference the perl code lose their relevance as soon as we ported them all and deleted the perl code.

Reply:
I'm not sure if we should encourage copying files from the host in general (see my later comment) and in general, this might not work with binary. We should inline the self.succeed in places where we really have to use it.

Reply:
Moreover this function is only used in the openssh test, so I'll remove the change completely.